support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -178,6 +179,10 @@ function buildRegularPaymentRequest(
 	const recaptchaToken = state.page.checkoutForm.recaptcha.token;
 	const promoCode = getPromoCode(promotions);
 	const giftRecipient = getGiftRecipient(state.page.checkoutForm.gifting);
+	// We will need to remove this !isProd check before we go live
+	const threeTierCreateSupporterPlusSubscription =
+		!isProd() &&
```

Comment:
Failsafe to stop creating products in production would guess?

Reply:
Correcto! Next step would be to see if I can allow test users or similar, but this would at least allow us to test e2e on `CODE`.